HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 ---
+## 1.5.0 / 2017-02-10
+- add `onDoubleClick` API
+- add `toggleSelect` option for changing how selection works
```

Comment:
I don't think you need this API, either.

If you want to disabled the default select behavior of TreeNode, just:

```jsx
<Tree selectedKeys={[]} /> 
```